components/NewsletterHome.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import { useRuntimeConfig } from '#app'
+import axios from 'axios'
+import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
+
+const config = useRuntimeConfig()
+
+const props = defineProps({
+  showNoThanks: {
+    type: Boolean,
+    default: false,
+  },
+  altDesign: {
+    type: Boolean,
+    default: false,
+  },
+  submitButtonText: {
+    type: String,
+    default: 'Sign up',
+  },
+  submitButtonIcon: {
+    type: String,
+    //default: 'pi-arrow-right',
+    default: null,
+  },
+})
+
+const emit = defineEmits<{
+  (e: 'submitEventEmit', value: any): void
+}>()
+
+const showComp = ref(true)
+const submissionStatus = ref(null)
+const isSubmiting = ref(false)
+
+// hides the entire component when the "No Thanks" button is clicked
+const hideComp = () => {
+  showComp.value = false
+}
+// submit the newsletter form and add email address to the Gothamist Newsletter list
+const submitForm = (email) => {
+  isSubmiting.value = true
+  submissionStatus.value = null
+  axios
+    .post(config.NEWSLETTER_API, {
+      list: '65dbec786b',
```

Comment:
This list id shouldn't be hardcoded here, we should put it in a runtime config variable. (If it doesn't change between demo and prod, we don't need to make a whole env var for it, just moving it to the config is fine)

Reply:
done